hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -189,4 +198,39 @@ def match_details(key=None) -> MatchDetailsResponse:
         results = get_match_details(dynamodb_table, key, image_folder_key)
         return MatchDetailsResponse(match_details=results)
 
+    @matches_api.post("/request-signal-opinion-change/")
+    def request_signal_opinion_change() -> ChangeSignalOpinionResponse:
+        """
+        request a change to the opinion for a signal in a dataset
+        """
+        signal_q = bottle.request.query.signal_q or None
+        signal_source = bottle.request.query.signal_source or None
+        dataset_q = bottle.request.query.dataset_q or None
+        opinion_change = bottle.request.query.opinion_change or None
+
+        if not signal_q or not signal_source or not dataset_q or not opinion_change:
```

Comment:
Finding this especially hard to read. Can this be done with
```python
if any(signal_q, signal_source, dataset_q, opinion_change) == False:
```
?

[Python's any function](https://docs.python.org/3/library/functions.html#any)

Reply:
Tried this change but mypy is not smart enough to understand `any` or `all` so I'm tempted to leave it or just break it up into multiple conditionals (I could use the walrus), thoughts?